aptos-move/framework/supra-framework/doc/supra_config.md

```diff
@@ -3,29 +3,25 @@
 
 # Module `0x1::supra_config`
 
-Maintains the consensus config for the blockchain. The config is stored in a
-Reconfiguration, and may be updated by root.
+Maintains protocol configuation settings specific to Supra. The config is stored in a
+may be updated by root.
```

Comment:
Something is amiss, it seems sentence is incomplete and grammatically incorrect.

Reply:
Updated!